src/content/blog/mastering-crypto.md

```diff
@@ -0,0 +1,76 @@
+
+A distributed ledger is a nice idea, but it is difficult to implement. In such peer-to-peer arrangements, security, synchronization, and consensus (whose version is correct) can become a developer's nightmare.
+
+Bitcoin's whitepaper proposed a unique database architecture to solve this problem: transactions are registered in discrete blocks, and no one can modify past blocks. To prevent alternative branches of blockchains, adding a new block requires a “proof-of-work” – a special CAPTCHA. This computation can adjust the difficulty to control the speed of block creation, ensuring a consistent and secure blockchain.
+
+Since the original Bitcoin blockchain idea, thousands of implementations and variations of the same **decentralized** ledger idea have occurred. (This suggests that the problem has actually not been solved.) Some blockchains are faster than others yet compromise on security; others are fast and secure yet work in small test environments and cannot be scaled.
+
+Nevertheless, they all have one thing in common: they contain and manage digital assets. Sometimes referred to as tokens, coins, cryptocurrency, crypto assets, or just **crypto**, these units can travel on their designated networks and help people send values between themselves.
+
+## Making Transactions with Crypto
+
+Let's begin with how to own cryptocurrency.
+
+It would be best to have a crypto wallet to receive, store, and send crypto. Unlike regular payment wallets such as PayPal, your assets are not tied to a specific wallet brand. Crypto wallets are similar to browsers, providing access to the blockchain where your crypto lives.
+
+However, crypto is always tied to the blockchain on which it originated. Each blockchain supports its own crypto set. Bitcoin blockchain has its native bitcoin (BTC), Ethereum has ether (ETH) and a whole universe of **tokens** anyone can create. SORA Network supports XOR, the network utility token, and other native tokens like VAL and PSWAP.
+
+Crypto wallets can support multiple blockchains. For each blockchain, users should create a separate **address (a public key)**. Users need **private keys** to access those. Think of it as your login and password. The public key (the address) can be shared with a counterparty, while private keys should be kept secret. For users’ convenience, public keys can be found in QR codes, while private keys are stored in a set of English words (your **secret phrase**).
+
+You can receive and store your crypto on your address on the blockchain (and use it with your securely stored private key). There are many ways to earn crypto, but the most common for a newbie is to buy it. **Crypto exchanges** convert your fiat currency (USD or EURO) to your preferred crypto. Once purchased, the exchange allows you to transfer the crypto to your address (public key).
+
+![using-qr](blog/mastering-crypto/using-qr.jpg)
```

Comment:
Path to the file should start with slash, so to be related to the root of the website and not to the current url

```
![using-qr](/blog/mastering-crypto/using-qr.jpg)
```